core/server/worker/src/test/java/alluxio/worker/block/UfsInputStreamManagerTest.java

```diff
@@ -177,10 +177,8 @@ public void testConcurrencyWithExpiration() throws Exception {
         threads.add(new Thread(runnable));
       }
       ConcurrencyUtils.assertConcurrent(threads, 30);
-      for (int i = 0; i < mNumOfInputStreams / 4; i++) {
-        // the first quarter of input streams are closed
-        Mockito.verify(mSeekableInStreams[i], Mockito.timeout(2000)).close();
-      }
+      // ensure at least one expired in stream is closed
+      Mockito.verify(mSeekableInStreams[0], Mockito.timeout(2000)).close();
```

Comment:
is this guaranteed to not be flaky?

Reply:
I think so. The entry is expected to expire after 20ms, so it should be evicted in the 2nd iteration of the loop.
If none input stream is closed, then there should be sth else wrong with the impl.

Reply:
why do we divide mNumOfInputStreams by 4?

Reply:
Replaced the constant 4 with the variable